draft-js-mention-plugin/src/MentionSuggestions/__test__/index.js

```diff
@@ -80,4 +80,47 @@ describe('MentionSuggestions Component', () => {
     });
     expect(suggestions.state().isActive).to.equal(false);
   });
+
+  it('The popoverComponent prop changes the popover component', () => {
+    const PopoverComponent = ({ children, ...props }) => {
+      return <div data-test-test {...props}>{children}</div>;
+    };
```

Comment:
eslint gives an odd error here:

    85:58  error  Unexpected block statement surrounding arrow body  arrow-body-style

IDK what's wrong with this... the code seems reasonably idiomatic to me.